R/create.R

```diff
@@ -275,17 +303,28 @@ create_stan_args <- function(model, data = NULL, init = "random",
   )
   
   # set up independent default arguments
-  if (method == "exact") {
+  if (method == "exact" && backend == "rstan") {
     default_args$cores <- 4
     default_args$warmup <- 500
     default_args$chains <- 4
     default_args$control <- list(adapt_delta = 0.99, max_treedepth = 15)
     default_args$save_warmup <- FALSE
     default_args$seed <- as.integer(runif(1, 1, 1e8))
+  }else if (method == "exact" && backend != "rstan"){
+    default_args$chains <- 4
+    default_args$seed <- as.integer(runif(1, 1, 1e8))
+    default_args$iter_warmup <- 500
+    default_args$iter_sampling <- ceiling(samples / 4)
+    default_args$parallel_chains <- 4
+    default_args$adapt_delta <- 0.99
+    default_args$max_treedepth <- 15
+    default_args$save_warmup <- FALSE
+    default_args$verbose <- FALSE
   }else if (method == "approximate") {
     default_args$trials <- 10
     default_args$iter <- 10000
     default_args$output_samples <- samples
+    default_args$seed <- as.integer(runif(1, 1, 1e8))
```

Comment:
This requires the following change (add `seed` argument) in `tests/testthat/test-create_stan_args.R` [line 14](tests/testthat/test-create_stan_args.R) to pass the checks:
```
-                                                                  "output_samples"))
+                                                                  "output_samples", "seed"))
```